dcat/rdf/dcat3.ttl

```diff
@@ -1,132 +1,152 @@
+# TO BE ADDED BEFORE PUBLICATION IN W3C SPACE
+#  dcterms:issued ""^^xsd:date ;
   dcterms:license <https://creativecommons.org/licenses/by/4.0/> ;
-  dcterms:modified "2012-04-24"^^xsd:date ;
-  dcterms:modified "2013-09-20"^^xsd:date ;
-  dcterms:modified "2013-11-28"^^xsd:date ;
-  dcterms:modified "2017-12-19"^^xsd:date ;
-  dcterms:modified "2019" ;
   dcterms:modified "2020-11-30"^^xsd:date ;
   dcterms:modified "2021-03-09"^^xsd:date ;
   dcterms:modified "2021-04-08"^^xsd:date ;
   dcterms:modified "2021-06-23"^^xsd:date ;
   dcterms:modified "2021-09-27"^^xsd:date ;
   dcterms:modified "2022-03-22"^^xsd:date ;
-  dcterms:modified "2022-05-12"^^xsd:date ;
+  dcterms:modified "2022-05-23"^^xsd:date ;
+  dcterms:modified "2023-01-05"^^xsd:date ;
+  dcterms:publisher <https://www.w3.org/> ;
+  dcterms:title "El vocabulario de catálogo de datos"@es ;
+  dcterms:title "Il vocabolario del catalogo dei dati"@it ;
+  dcterms:title "Le vocabulaire des jeux de données"@fr ;
+  dcterms:title "Slovník pro datové katalogy"@cs ;
+  dcterms:title "The data catalog vocabulary"@en ;
+  dcterms:title "Το λεξιλόγιο των καταλόγων δεδομένων"@el ;
+  dcterms:title "أنطولوجية فهارس قوائم البيانات"@ar ;
+  dcterms:title "データ・カタログ語彙（DCAT）"@ja ;
+  dcterms:title "Datakatalogvokabular"@da ;
+  foaf:depiction <https://www.w3.org/TR/vocab-dcat-3/images/dcat-all-attributes.svg> ;
+  owl:backwardCompatibleWith <http://www.w3.org/ns/dcat2> ;
+  owl:backwardCompatibleWith <http://www.w3.org/ns/dcat2014> ;
+  owl:imports dcterms: ;
+  owl:imports <http://www.w3.org/2004/02/skos/core> ;
+  owl:imports <http://www.w3.org/ns/prov-o#> ;
+  owl:priorVersion <http://www.w3.org/ns/dcat2> ;
+  owl:versionInfo "3" ;
+  owl:versionIRI <http://www.w3.org/ns/dcat3> ;
   rdfs:comment "DCAT es un vocabulario RDF diseñado para facilitar la interoperabilidad entre catálogos de datos publicados en la Web. Utilizando DCAT para describir datos disponibles en catálogos se aumenta la posibilidad de que sean descubiertos y se permite que las aplicaciones consuman fácilmente los metadatos de varios catálogos."@es ;
   rdfs:comment "DCAT est un vocabulaire développé pour faciliter l'interopérabilité entre les jeux de données publiées sur le Web. En utilisant DCAT pour décrire les jeux de données dans les catalogues de données, les fournisseurs de données augmentent leur découverte et permettent que les applications facilement les métadonnées de plusieurs catalogues. Il permet en plus la publication décentralisée des catalogues et facilitent la recherche fédérée des données entre plusieurs sites. Les métadonnées DCAT aggrégées peuvent servir comme un manifeste pour faciliter la préservation digitale des ressources. DCAT est définie à l'adresse http://www.w3.org/TR/vocab-dcat/. Une quelconque version de ce document normatif et ce vocabulaire est une erreur dans ce vocabulaire."@fr ;
```

Comment:
the same changes as suggested above (dcterms:descriptions) should be applied here as well

Reply:
```suggestion
  rdfs:comment "DCAT est un vocabulaire développé pour faciliter l'interopérabilité entre les jeux de données publiées sur le Web. En utilisant DCAT pour décrire les jeux de données dans les catalogues de données, les fournisseurs de données facilitent leur découverte et permettent que les applications consomment facilement les métadonnées de plusieurs catalogues. Il permet de plus la publication décentralisée des catalogues et facilite la recherche fédérée des données entre plusieurs sites. Les métadonnées DCAT aggrégées peuvent servir comme un manifeste pour faciliter la préservation digitale des ressources. DCAT est définie à l'adresse http://www.w3.org/TR/vocab-dcat/. Toute différence entre ce document normatif et le présent vocabulaire est une erreur dans le vocabulaire."@fr ;
```